examples/templates/Example.js

```diff
@@ -206,6 +217,32 @@
             if ( optionalArgs )
                 osg.extend( optionsDebug, optionalArgs );
 
+            if ( !this._programRTT ) {
```

Comment:
"texture are 0 so textureNull"
Not sure to understand, code does
"stateSet.setTextureAttributeAndModes( 0, texture );"
So no null nor texturenull here ?

It does set the texture (otherwise special shader wouldn't work?).
Withtout special shader, we get black quad as shader compiler outputs "shaders with shadows"
even with the debugNodeRTT isn't under the shadowed scene node, the one with shadow receiver attributes.

I dont want you workaround stuff if you dont understand why it does not work.
It's code from old shadow demo from master, already reviewed once, that's why I didn't think twice about spending time to double question and digg it.
( I think we even did had a discussion about that exact thing, shader compiler getting attributes default of material/light/etc and doing a "material" shader where it shouldn't as debugNode is a sibling of the node containing shadowedscene )
I can dig more and make a repro case easier to debug ?